docs/LogGuide.md

```diff
@@ -760,26 +763,33 @@ to include both possibilities.
 
 Ground AOEs that don't hit anybody are considered [NetworkAOEAbility](#line22) lines.
 
+There are two fields on 21/22 lines that provide information about the number of targets affected. The `targetCount`
+field indicates the number of targets. The `targetIndex` field indicates which target this particular line refers to.
+For example, for a 21-line, you would see a `targetIndex` of 0, and a `targetCount` of 1. For an AoE ability that hits
```

Comment:
You should only use a hyphen on compound nouns that are being used as an adjective, e.g. two-party system or 10-minute enrage.  "21-line" isn't being used as an adjective and so shouldn't have a hyphen, in my opinion.  You do this in a bunch of other places in this PR, which I haven't listed.

```suggestion
For example, for a 21 line, you would see a `targetIndex` of 0, and a `targetCount` of 1. For an AoE ability that hits
```